vendor/engines/projects/app/views/projects/projects/show.html.haml

```diff
@@ -2,4 +2,4 @@
   = @project.facility
 
 %h2= @project.name
-%p= @project.description
+%p= @project.description.html_safe
```

Comment:
Since `description` will already have the `<p>` tag, do we want it here?

Also, I'm wondering if using the `sanitize` helper method might be useful (probably apply it elsewhere as well). After a quick test, we would need to add `style` to the list of allowed attributes.

I think we could do that in an initializer
`ActionView::Base.sanitized_allowed_attributes << "style"`

http://apidock.com/rails/v3.2.13/ActionView/Helpers/SanitizeHelper/sanitize

What do you think?